### DISCUSSION TITLE
zsh: no such file or directory after installing Homebrew

### DISCUSSION THREAD

**Discussion by selinacmh** [Homebrew/discussions#2616] [Getting started] [answered]
After installing Homebrew, i couldn't run the two commands which add Homebrew to my path, terminal shows 'no such file or directory'. I have tried renaming the .zprofile (found this in Apple communities) and it didn't work. What should I do to continue? Thank you.

==> Next steps:
- Run these two commands in your terminal to add Homebrew to your PATH:
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/selinahui/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

selinahui@Selinas-Air ~ % echo 'eval "$(/opt/homebrew/bin/brew shellenv)"'
eval "$(/opt/homebrew/bin/brew shellenv)"
selinahui@Selinas-Air ~ % /Users/selinahui/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
zsh: no such file or directory: /Users/selinahui/.zprofile

**Comment by carlocab** [comment, marked as answer]
It looks like you ran only

```
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"'
```

You need to run

```
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/selinahui/.zprofile
```

**Reply by selinacmh** [reply]
oh no...im so stupid...thank you

**Reply by joshuaoliverio** [reply]
Hi @carlocab ! I followed these steps. But then, I decided to remove homebrew. I ran the uninstall command and it worked. But now, every time I open terminal it says "/Users/joshuaoliverio/.zprofile:1: no such file or directory: /opt/homebrew/bin/brew
/Users/joshuaoliverio/.zprofile:2: no such file or directory: /opt/homebrew/bin/brew"

How do I fix this?

Thanks!

**Reply by carlocab** [reply]
https://github.com/Homebrew/discussions/discussions/3446#discussioncomment-3184688
